Doppler.ContactPolicies.Business.Logic/UserApiClient/Services/IUserFeaturesService.cs

```diff
@@ -0,0 +1,9 @@
+using System.Threading.Tasks;
+
+namespace Doppler.ContactPolicies.Business.Logic.UserApiClient.Services
+{
+    public interface IUserFeaturesService
+    {
+        Task<bool> GetUserContactPoliciesFeatureAsync(string accountName);
```

Comment:
Sería un nombre mas descriptivo? 

```suggestion
        Task<bool> HasContactPoliciesFeatureAsync(string accountName);
```